flake.nix

```diff
@@ -0,0 +1,68 @@
+{
+  description = "Ontime is a browser-based application that manages event rundowns, scheduling, and cueing.";
+
+  inputs = {
+    flake-parts.url = "github:hercules-ci/flake-parts";
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+  };
+
+  outputs =
+    inputs@{ flake-parts, ... }:
+    flake-parts.lib.mkFlake { inherit inputs; } {
+      systems = [
+        "x86_64-linux"
+        "aarch64-linux"
+        "aarch64-darwin"
+        "x86_64-darwin"
+      ];
+      perSystem =
+        {
+          config,
+          self',
+          inputs',
+          pkgs,
+          system,
+          ...
+        }:
+        {
+          formatter = pkgs.nixfmt-tree;
+          packages.default = pkgs.callPackage (
```

Comment:
I'd do `callPackage ./package.nix {}` and put the package in there, because better seperation of concerns and compatibility with non-flake nix (provided you also create a default.nix that also `callPackage`s)